pkg/operator2/routercerts/controller.go

```diff
@@ -197,20 +198,20 @@ func verifyWithAnyCertificate(serverCerts []*x509.Certificate, options x509.Veri
 	return err
 }
 
-func (c *RouterCertsDomainValidationController) Run(workers int, stopCh <-chan struct{}) {
+func (c *RouterCertsDomainValidationController) Run(ctx context.Context, workers int) {
```

Comment:
Please move the non-automatic changes outside the bump commit

Reply:
Created a new commit for these changes